src/utils/Utils.ts

```diff
@@ -592,6 +593,25 @@ class Utils {
         }
     }
 
+    static getNormalizedFunctionName(request: any, lambdaArn?: string) {
```

Comment:
I think it might be better to have `LambdaUtils` and move `getNormalizedFunctionName` to there. Otherwise, with the new platform supports, `Utils` might be messed up

Reply:
@serkan-ozal makes sense. I'll do the appropriate changes.